packages/arb-token-bridge-ui/src/util/__tests__/NumberUtils.test.ts

```diff
@@ -0,0 +1,144 @@
+import { BigNumber } from 'ethers'
+import { formatAmount } from '../NumberUtils'
+
+describe('formatAmount', () => {
+  describe('for short token symbol', () => {
+    const getResultFromBigNumber = (value: string) =>
+      formatAmount(BigNumber.from(value), { decimals: 6, symbol: 'ETH' })
```

Comment:
was confused by this test case because ETH has 18 decimals right?
Can we use a random symbol or use 18 for ETH?
please correct me if i'm wrong

Reply:
Yes, ETH has 18 decimals, you're right. I'll update it to some random symbol, otherwise I would need to add 12 numbers to to every call of this function